src/WebServerManager.cpp

```
@@ -363,84 +363,64 @@ void WebServerManager::handleApiRegisterUid() {
+        serializeJson(spoolBody, spoolJson);
+        Serial.printf("register-uid: creating spool: %s\n", spoolJson.c_str());
+
+        snprintf(url, sizeof(url), "%s/api/v1/spool", baseUrl);
+        http.begin(client, url);
+        http.addHeader("Content-Type", "application/json");
+        code = http.POST(spoolJson);
+        if (code != 200 && code != 201) {
+            response = http.getString();
+            http.end();
+            Serial.printf("register-uid: spool creation failed HTTP %d: %s\n", code, response.c_str());
+            String errMsg = "Failed to create spool (HTTP " + String(code) + "): " + response;
```

Clang (14.0.6)

[warning] variable 'errMsg' is not initialized

(cppcoreguidelines-init-variables)